cmd/saturn-l2/main.go

```diff
@@ -114,6 +116,12 @@ var (
 	saturn_l1_hostName = "saturn-test.network"
 
 	defaultL1DiscoveryURL = "https://orchestrator.saturn-test.network/nodes/nearby"
+
+	checkL1ConnectivityInterval = 5 * time.Second
```

Comment:
@bajtos Please feel free to change this to a number that makes sense from a UX perspective

Reply:
5 seconds is good for starters 👍🏻 Let's see how it works in practice.